modules/farm_fd2/src/entrypoints/transplanting/lib.submit.unit.cy.js

```diff
@@ -0,0 +1,410 @@
+      );
+
+      expect(rowsPerBedQuantity.relationships.inventory_asset).to.be.null;
+    });
+  });
+
+  it('Check the row feet quantity--standard', () => {
+    cy.wrap(
+      farmosUtil.getStandardQuantity(result.transplantingRowFeetQuantity.id)
+    ).then((rowFeetQuantity) => {
+      expect(rowFeetQuantity.type).to.equal('quantity--standard');
+      expect(rowFeetQuantity.attributes.measure).to.equal('length');
+      expect(rowFeetQuantity.attributes.value.decimal).to.equal(
+        (form.bedFeet * form.rowsPerBed).toString()
+      );
+      expect(rowFeetQuantity.attributes.label).to.equal('Row Feet');
+
+      expect(rowFeetQuantity.attributes.inventory_adjustment).to.equal(
+        'increment'
+      );
+
+      expect(rowFeetQuantity.relationships.units.type).to.equal(
+        'taxonomy_term--unit'
+      );
+      expect(rowFeetQuantity.relationships.units.id).to.equal(
+        result.transplantingRowFeetQuantity.relationships.units.id
+      );
+
+      expect(rowFeetQuantity.relationships.inventory_asset.type).to.equal(
+        'asset--plant'
+      );
+      expect(rowFeetQuantity.relationships.inventory_asset.id).to.equal(
+        result.transplantingRowFeetQuantity.relationships.inventory_asset.id
+      );
+    });
+  });
+
+  it('Check the bed width quantity--standard', () => {
+    cy.wrap(
+      farmosUtil.getStandardQuantity(result.transplantingBedWidthQuantity.id)
+    ).then((bedWidthQuantity) => {
+      expect(bedWidthQuantity.type).to.equal('quantity--standard');
+      expect(bedWidthQuantity.attributes.measure).to.equal('length');
+      expect(bedWidthQuantity.attributes.value.decimal).to.equal(
+        form.bedWidth.toString()
+      );
+      expect(bedWidthQuantity.attributes.label).to.equal('Bed Width');
+
+      expect(bedWidthQuantity.attributes.inventory_adjustment).to.be.null;
+
+      expect(bedWidthQuantity.relationships.units.type).to.equal(
+        'taxonomy_term--unit'
+      );
+      expect(bedWidthQuantity.relationships.units.id).to.equal(
+        result.transplantingBedWidthQuantity.relationships.units.id
+      );
+
+      expect(bedWidthQuantity.relationships.inventory_asset).to.be.null;
+    });
+  });
+
+  it('Check the log--activity', () => {
+    cy.wrap(
+      farmosUtil.getTransplantingActivityLog(result.transplantingLog.id)
+    ).then((transplantingLog) => {
+      expect(transplantingLog.type).to.equal('log--activity');
+      expect(transplantingLog.attributes.name).to.equal(
+        '1950-01-02_xp_BROCCOLI'
+      );
+      expect(transplantingLog.attributes.timestamp).to.contain('1950-01-02');
+      expect(transplantingLog.attributes.status).to.equal('done');
+      expect(transplantingLog.attributes.is_movement).to.be.true;
+
+      expect(transplantingLog.relationships.location.length).to.equal(3);
+      expect(transplantingLog.relationships.location[0].id).to.equal(
+        result.transplantingLog.relationships.location[0].id
```

Comment:
Use `form.location` and the `fieldMap` to get the `id` of the field that should have been put into the database.

Similarly for the beds below.